documentation/public/user-guide/introduction.md

```diff
@@ -0,0 +1,15 @@
+## What is Slang?
+
+Slang is intended to be a modular Solidity compiler, specifically targeting code analysis and developer tooling. This means servicing tools with domain-specific APIs and, in general, facilitating working with and analyzing the Solidity source code. If you're in the editor writing Solidity or performing linting or additional validation, there's a chance that you are, or could be, running Slang!
+
+## What Slang is not?
+
+First and foremost, it is not a replacement for `solc`, the standard Solidity compiler. We do not plan at the moment to support emitting EVM bytecode. Secondly, it is not meant to be used for formal verification of contracts or Solidity logic in general.
+
+## Supporting multiple versions
+
+The Solidity programming language has evolved quite a bit since its inception. Some features were introduced, some changed, while some eventually became obsolete and were removed altogether.
+
+While it's good for a programming language to evolve and better serve the needs of its users, not being able to easily upgrade or re-deploy existing contracts poses a unique challenge. Older contracts that are still being used on the blockchain, written in older versions of Solidity, must remain understandable during the development process.
+
```

Comment:
Suggestion to add a paragraph at the end:

> This is why, instead of downloading a different Slang version for each Solidity version you want to use, you only need to use the latest Slang version, and initialize the Slang language API with the Solidity version instead. See [language specific guides](link) for more examples.

Reply:
Added, thanks! I've adapted the sentence to not repeat Solidity/Slang as much, hopefully it's still as clear.